fbchat/graphql.py

```diff
@@ -118,6 +118,26 @@ def graphql_to_group(group):
         message_count=group.get('messages_count')
     )
 
+def graphql_to_room(room):
+    if room.get('image') is None:
+        room['image'] = {}
+    c_info = get_customization_info(room)
+    return room(
```

Comment:
`Room`, not `room` ;)

Reply:
How are you testing?
It didn't appear when I tested this :(